Plugins/Flow.Launcher.Plugin.Program/Programs/UWP.cs

```diff
@@ -320,7 +321,27 @@ public Result Result(string query, IPublicAPI api)
                     ContextData = this,
                     Action = e =>
                     {
-                        Launch(api);
+                        var elevated = (
+                            e.SpecialKeyState.CtrlPressed &&
+                            e.SpecialKeyState.ShiftPressed &&
+                            !e.SpecialKeyState.AltPressed &&
+                            !e.SpecialKeyState.WinPressed
+                        );
+
+                        if (elevated)
+                        {
+                            if (!CanRunElevated)
+                            {
+                                return false;
```

Comment:
Maybe notify user that this app cannot be run as admin (and probably launch it as normal user).

Reply:
done